utils/send_email.py

```diff
@@ -1,34 +1,90 @@
+from django.conf import settings
 from django.core.mail import send_mail
-from supplier_app.models import Company, CompanyUserPermission
 from django.template.loader import render_to_string
 from django.utils.html import strip_tags
 
+from supplier_app.models import (
+    Company,
+    CompanyUserPermission,
+)
+from supplier_app import email_notifications
+
 
 def send_email_notification(subject, message, recipient_list):
     plain_message = strip_tags(message)
     send_mail(
         subject,
         plain_message,
-        'Suppliers Management Eventbrite <jx3.team@gmail.com>',
+        'Suppliers Management Eventbrite <{}>'.format(
+            settings.EMAIL_HOST_USER
+        ),
         recipient_list,
         html_message=message,
-        fail_silently=False,
```

Comment:
This will raise an exception in case of failure, are we managing this?

Reply:
we removed it from there and we are going to handle failure in the views in this PR (https://github.com/evbeda/suppliers_management/pull/134) we are working on that.